src/features/DisplayControls/NursingTasks/components/TaskTile.jsx

```diff
@@ -43,19 +43,32 @@ export default function TaskTile(props) {
     isDisabled,
     administeredTimeInEpochSeconds,
     serviceType,
+    isANonMedicationTask,
+    creator,
   } = newMedicationNursingTask;
 
   const isRelevantTask = getRelevantTaskStatus(
     startTimeInEpochSeconds,
     nursingTasks
   );
 
+  const capitalizeWords = (name) => {
+    return name.replace(/\b\w/g, function (char) {
+      return char.toUpperCase();
+    });
+  };
```

Comment:
we can use css text-transform property for uppercase instead of JS

Reply:
But if text transform is used the whole text will be in upper case . We only want the first letter of the name in uppercase

Reply:
we have text-transform: capitalize; for the same thing